### PR TITLE
Define each word in the custom forest.dic dictionary 

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes https://github.com/ChainSafe/forest/issues/1854

**Other information and links**

### CODE REVIEW

File: documentation/developer_documentation/glossary.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Merkle
+
+[Ralph C. Merkle](https://en.wikipedia.org/wiki/Ralph_Merkle) is a computer scientist. He is one of the inventors of public-key cryptography, the inventor of cryptographic hashing. Merkle tree is named after him.
+
+## IPFS
+[IPFS](https://github.com/ipfs/ipfs) stands for InterPlanetary File System which is A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+
+## Proof of Spacetime (PoSt)
+
+[_Proof-of-Spacetime_](pos#post) is a procedure by which a [storage-miner](glossary#storage-miner-actor) can prove to the Filecoin network they have stored and continue to store a unique copy of some data on behalf of the network for a period of time. Proof-of-Spacetime manifests in two distinct varieties in the present Filecoin specification: [Window Proof-of-Spacetime](glossary#window-proof-of-spacetime-windowpost) and [Winning Proof-of-Spacetime](glossary#winning-proof-of-spacetime-winningpost).
```
Comment (LesnyRumcajs) [inline documentation/developer_documentation/glossary.md:58]: all of the links in this section are broken

File: documentation/developer_documentation/glossary.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+A set of CIDs forming a unique key for a tipset.
+
+## mempool
+
+mempool stands for the Message Pool, which is the component of forest that handles pending messages for inclusion in the chain. Messages are added either directly for locally published messages or through pubsub propagation.
+
+## Merkle
+
+[Ralph C. Merkle](https://en.wikipedia.org/wiki/Ralph_Merkle) is a computer scientist. He is one of the inventors of public-key cryptography, the inventor of cryptographic hashing. Merkle tree is named after him.
```
Comment (LesnyRumcajs) [inline documentation/developer_documentation/glossary.md:51]: Best to provide a link to Merkle tree article, it's probably more interesting to someone looking into the glossary.

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]

File: documentation/developer_documentation/glossary.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+## mempool
+
+mempool stands for the Message Pool, which is the component of forest that handles pending messages for inclusion in the chain. Messages are added either directly for locally published messages or through pubsub propagation.
+
+## Merkle
+
+[Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree) is a tree in which every node is labelled with the cryptographic hash of a data block, and every node that is not a leaf (called a branch, inner node, or inode) is labelled with the cryptographic hash of the labels of its child nodes. A hash tree allows efficient and secure verification of the contents of a large data structure.
+
+## IPFS
+[IPFS](https://github.com/ipfs/ipfs) stands for InterPlanetary File System which is A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
```
Comment (LesnyRumcajs) [inline documentation/developer_documentation/glossary.md:54]:
```suggestion
[IPFS](https://github.com/ipfs/ipfs) stands for InterPlanetary File System which is a peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
```

**Comment by lemmih** [comment]
@tyshko5 Looks good. Could you update `SUMMARY.md` to include a link to the glossary?

**Review by lemmih — CHANGES_REQUESTED** [review]

File: documentation/src/SUMMARY.md [hunk 4]
```diff
@@ -4,3 +4,4 @@
 - [Basic Usage](./basic_usage.md)
 - [Configuration](./configuration.md)
 - [Trouble Shooting](./trouble_shooting.md)
+- [Glossary](../developer_documentation/glossary.md)
```
[the remark this answers is not in the document]
Comment (tyshko5) [reply documentation/src/SUMMARY.md]: Fixed

**Review by lemmih — CHANGES_REQUESTED** [review]
Great. We only need one copy of the glossary, though.

**Review by lemmih — APPROVED** [review]

**Pull request merged**